Notepad/Element.swift

```diff
@@ -22,7 +22,7 @@ public enum Element: String {
     case bold = "(^|[\\W_])(?:(?!\\1)|(?=^))(\\*|_)\\2(?=\\S)(.*?\\S)\\2\\2(?!\\2)(?=[\\W_]|$)"
     case italic = "(^|[\\W_])(?:(?!\\1)|(?=^))(\\*|_)(?=\\S)((?:(?!\\2).)*?\\S)\\2(?!\\2)(?=[\\W_]|$)"
     case boldItalic = "(\\*\\*\\*\\w+(\\s\\w+)*\\*\\*\\*)"
-    case code = "(`.*`)" //Allows for any character to be in incline code.
+    case code = "(`[^`]{1,}`)" //Allows for any character except ` to be in incline code.
```

Comment:
Looks good! Can you also fix "incline"/"inline" typo and add a space after `//`?